readme.md

```diff
@@ -86,19 +86,35 @@ filter(is(Number), [0, '1', 2, null]); // [0, 2]
 
 ## Partial Application
 
-The process of getting a function with lesser arity compared to the original
-function by fixing the number of arguments is known as partial application.
+Partially applying a function means creating a new function by pre-filling some of the arguments to the original function.
```

Comment:
The arguments aren't broken, why fix 'em? ;)

On Mon, Jul 25, 2016, 10:42 PM hemanth.hm <notifications@github.com> wrote:

> In readme.md
> <https://github.com/hemanth/functional-programming-jargon/pull/75#discussion_r72190688>
> :
>
> > @@ -86,19 +86,35 @@ filter(is(Number), [0, '1', 2, null]); // [0, 2]
> >
> >  ## Partial Application
> >
> > -The process of getting a function with lesser arity compared to the original
> > -function by fixing the number of arguments is known as partial application.
> > +Partially applying a function means creating a new function by pre-filling some of the arguments to the original function.
>
> fixing the number of arguments
>
> vs
>
> pre-filling some of the arguments
>
> 🤔
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hemanth/functional-programming-jargon/pull/75/files/7027787ffbc2716fbb65a1e081967ae61d95a2a7#r72190688>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAB-4MKhucqULPaKpfEdlrE5ABchRfDfks5qZZ5TgaJpZM4JUs_Y>
> .
>